Source/UETorch/Private/TorchContext.cpp

```diff
@@ -81,11 +83,53 @@ bool FTorchUtils::CallFunctionString(lua_State* LuaState, const ANSICHAR* Functi
 	const int NumResults = 1;
 	if (lua_pcall(LuaState, NumArgs, NumResults, 0) != 0)
 	{
-		UE_LOG(LogScriptPlugin, Warning, TEXT("Cannot call Lua function %s: %s"), ANSI_TO_TCHAR(FunctionName), ANSI_TO_TCHAR(lua_tostring(LuaState, -1)));
+		UE_LOG(LogUETorch, Warning, TEXT("Cannot call Lua function %s: %s"), ANSI_TO_TCHAR(FunctionName), ANSI_TO_TCHAR(lua_tostring(LuaState, -1)));
+		bResult = false;
+	}
+	if (!lua_isstring(LuaState, -1)) {
+		UE_LOG(LogUETorch, Warning, TEXT("Lua function %s did not return a string"), ANSI_TO_TCHAR(FunctionName));
```

Comment:
I suggest here (and in my original function) we also accept nil, and just set Out to '' or '<nil>'. Often we don't want to return a result.